### DISCUSSION TITLE
Image previews in DBeaver

### DISCUSSION THREAD

**Discussion by sovereignstack** [dbeaver/dbeaver#36635] [Ideas]
I have image URLs stored in a table. I would like DBeaver to show me previews of those images.  Like the rows can be taller and it only needs to load the images present on the current page - like 10-15 max at a time. It just has to do a HTTP GET on those URLs.
I am dabbling in vector search and vision models and I want to see the effectiveness of various models. So when I search beach photos I want to see if there are just beach photos or photos with a pool background too. It would be great if DBeaver supports this. That way I won't have to build a UI just for such quick testing.

**Comment by serge-rider** [comment]
It can be easily done with column value transformation. 
https://dbeaver.com/docs/dbeaver/Data-View-and-Format/#transforming-data-presentation

Thus you can view pages in the embedded browser even column contains only a part of URL (e.g. image ID)
<img width="683" alt="image" src="https://github.com/user-attachments/assets/06c827ff-22a8-4d9f-bd7b-e394193e5427" />
